CvGameCoreDLL/CvGameTextMgr.cpp

```diff
@@ -6492,6 +6492,19 @@ void CvGameTextMgr::parseSpecialistHelpActual(CvWStringBuffer &szHelpString, Spe
 			}
 			bStarted = setResumableCommerceTimes100ChangeHelp(szHelpString, szStart, L": ", L"", aiCommerces, true, bStarted);
 
+			// Happiness
+			if (iHappinessChange != 0)
+			{
+				if (iHappinessChange > 0)

```

Comment:
You can remove that `if` statement and collapse the two `setResumableValueChangeHelp` calls (see other examples that display happiness), something like `setResumableValueChangeHelp(szHelpString, szStart, L": ", L"", std::abs(iHappinessChange), iHappiness > 0 ? gDLL->getSymbolID(HAPPY_CHAR) : gDLL->getSymbolID(UNHAPPY_CHAR), false, true, bStarted)`